scripts/install.sh

```diff
@@ -25,3 +25,11 @@ if [[ "${TOX_ENV}" == "pypy" ]]; then
     pyenv install pypy-2.6.0
     pyenv global pypy-2.6.0
 fi
+
+if [[ "${TOX_ENV}" == "gae" && ! -d $HOME/google_appengine ]]; then
+    cd $HOME
+    wget https://storage.googleapis.com/appengine-sdks/featured/google_appengine_1.9.25.zip
+    unzip -q google_appengine_1.9.25.zip
+    # Directory is required for the write to well known file test to pass.
+    mkdir -p $HOME/.config/gcloud
```

Comment:
I disagree. We should just monkey patch the `os.getenv` that fails in the `gae` unit tests.